internal/controllers/project/actuator.go

```diff
@@ -128,10 +131,106 @@ func (actuator projectActuator) CreateResource(ctx context.Context, obj orcObjec
 	return osResource, nil
 }
 
-func (actuator projectActuator) DeleteResource(ctx context.Context, _ orcObjectPT, project *projects.Project) progress.ReconcileStatus {
+func (actuator projectActuator) DeleteResource(ctx context.Context, _ orcObjectPT, project *osResourceT) progress.ReconcileStatus {
 	return progress.WrapError(actuator.osClient.DeleteProject(ctx, project.ID))
 }
 
+func (actuator projectActuator) GetResourceReconcilers(ctx context.Context, orcObject orcObjectPT, osResource *osResourceT, controller generic.ResourceController) ([]resourceReconciler, progress.ReconcileStatus) {
+	return []resourceReconciler{
+		actuator.updateResource,
+	}, nil
+}
+
+func (actuator projectActuator) updateResource(ctx context.Context, obj orcObjectPT, osResource *osResourceT) progress.ReconcileStatus {
+	log := ctrl.LoggerFrom(ctx)
+	resource := obj.Spec.Resource
+	if resource == nil {
+		return progress.WrapError(
+			orcerrors.Terminal(orcv1alpha1.ConditionReasonInvalidConfiguration, "Update requested, but spec.resource is not set"))
+	}
+
+	var updateOpts projects.UpdateOpts
+
+	handleNameUpdate(&updateOpts, obj, osResource)
+	handleDescriptionUpdate(&updateOpts, resource, osResource)
+	handleEnabledUpdate(&updateOpts, resource, osResource)
+	// FIXME(dkokkino): Unlike the networkClient, which supports tag updates for controllers
+	// like subnets, ports, networks, and security groups, the identityClient doesn't support
+	// tag updates via the ReconcileTags method. Using a custom handler to update tags as a workaround for now.
```

Comment:
The reason for that is simple: Neutron uses a [tag extension](https://docs.openstack.org/api-ref/network/v2/#tag-extension-tags) and doesn't allow adding tags at creation time (we need extra API calls for setting or updating tags), while keystone has [support for tags builtin](https://docs.openstack.org/api-ref/identity/v3/#create-project) and we can set tags in the same API call as the resource creation or update.

We can drop the comment IMO.